### PR TITLE
Add bcftools csi and tbi outputs

### PR DESCRIPTION
**Pull request by fellen31**
This PR adds optional `csi` and `tbi` outputs to all *nf-test* bcftools modules. 
Adds normal and  stub tests for `--write-index`, `--write-index=tbi` and `--write-index=csi`.

## PR checklist 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Review by nvnieuwk — APPROVED** [review]
This LGTM and will help in simplifying a lot of pipelines, thank you! Can you wait for a second review here since this will impact a lot of pipelines?

**Comment by fellen31** [comment]
Nice! I'll wait for a second one.

**Review by maxulysse — APPROVED** [review]

**Pull request merged**
